app/src/components/integrations/elements/pluginIcon/pluginIcon.jsx

```diff
@@ -44,7 +49,9 @@ export const PluginIcon = ({ pluginData, className, ...rest }) => {
         return null;
       }
 
-      return URLS.projectIntegrationByIdCommand(projectId, integration.id, COMMAND_GET_FILE);
+      return isPublic
+        ? URLS.pluginPublicFile(integration.name, FILE_KEY)
+        : URLS.projectIntegrationByIdCommand(projectId, integration.id, COMMAND_GET_FILE);
     }
```

Comment:
We have several cases when the plugin will try to load its icon via API:
1) Plugin supports `commonCommand`
2) Plugin supports integration based command
3) Plugin public and have to load icon via `URLS.pluginPublicFile`
4) Plugin private and have to load icon via `URLS.pluginFile`

These 4 cases should be supported until the 1st and 2nd options will be removed as legacy.

Reply:
How to check if plugin supports integration based command ? Because now during checking looks like Sign Up supports it, but it causes error